vim_llm_stream.py

```diff
@@ -0,0 +1,26 @@
+import pynvim
```

Comment:
Thanks! Can you register the plugin with a bit of vimscript? Also I think you should use feedkeys instead of input. Not 100% sure what the difference is.